### DISCUSSION TITLE
Caso d'uso Trascr_021: dati sentenza tribunale italiano

### DISCUSSION THREAD

**Discussion by antonioluberto** [italia/ansc#942] [Supporto servizi cooperativi]
La documentazione relativa al caso d'uso in oggetto prevede attualmente la sezione (obbligatoria):

![image](https://github.com/italia/ansc/assets/26415430/473acb7a-63a2-4586-9006-4d5b2ea04931)

Trattandosi però di richiesta dall'autorità diplomatica o consolare, il Comune procede alla redazione in ragione della sola documentazione ricevuta da ambasciata/consolato (che, peraltro, richiede la trascrizione del provvedimento estero al Comune solo nel caso in cui il provvedimento medesimo non richieda sentenza di Tribunale italiano che ne sancisca la validità in Italia).
Ci pare dunque che questa sezione debba essere rimossa.
Si fa notare che logica analoga esiste già per altri casi d'uso quali, ad esempio, Trascr_009

**Comment by asqueglia** [comment]
Buongiorno, abbiamo sottoposto il quesito ad ANUSCA. Di seguito la risposta

ANUSCA conferma che i casi particolari si riferiscono solo ai minorenni. Dovrebbe essere l’interessato che chiede la trascrizione della sentenza del tribunale italiano; se proviene da consolato non è obbligatoria la sentenza del tribunale italiano.
Quindi procederemo a rendere opzionale la sezione.

**Comment by asqueglia** [comment]
Buonasera con la versione 1.32 è stata corretta la segnalazione. Chiudiamo pertanto la issue
